deploy/crds/hco_v1alpha1_hyperconverged_crd.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: hyperconvergeds.hco.kubevirt.io
+spec:
+  group: hco.kubevirt.io
```

Comment:
Maybe the group should be just `kubevirt.io` or is there some requirement that these be `hyperconverged.hco.kubevirt.io`?

Reply:
Take that back, rbac rules is where there needs to be significance.  Having the same group will make the hco share rules with all kubevirt apis.  Not sure we want to do that just yet.  

I don't think there's harm in leaving as is, so we can revisit later.